lighthouse-core/report/v2/renderer/performance-category-renderer.js

```diff
@@ -45,13 +45,13 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
    * @return {!Element}
    */
   _renderPerfHintAudit(audit, scale) {
-    const extendedInfo = /** @type {!PerformanceCategoryRenderer.PerfHintExtendedInfo}
-        */ (audit.result.extendedInfo);
+    const summaryInfo = /** @type {!PerformanceCategoryRenderer.PerfHintSummaryInfo}
+        */ (audit.result.summary);
```

Comment:
I might be missing something, I thought we were nesting `summary` inside of `details`?